core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -3560,7 +3560,8 @@ argv_environ_init(struct argv_environ_values *argv_environ, char *argv_buf,
 
 void
 argv_environ_destroy(struct argv_environ_values *argv_environ)
-{}
+{
```

Comment:
Should not change here, or the CI may report code guideline check error:
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/6088777667/job/16520150912